Looking in indexes: https://pypi.org/simple, http://dvlp1.geo.tuwien.ac.at:8080/simple
Requirement already satisfied: cadati in c:\users\wpreimes\appdata\local\continuum\miniconda2\envs\pytesmo\lib\site-packages\cadati-0.0.1.post0.dev2+gfebbc8e-py3.6.egg (0.0.1.post0.dev2+gfebbc8e)
Requirement already satisfied: numpy in c:\users\wpreimes\appdata\local\continuum\miniconda2\envs\pytesmo\lib\site-packages (from cadati) (1.18.4)
Requirement already satisfied: scipy in c:\users\wpreimes\appdata\local\continuum\miniconda2\envs\pytesmo\lib\site-packages (from cadati) (1.3.1)
Requirement already satisfied: pandas in c:\users\wpreimes\appdata\local\continuum\miniconda2\envs\pytesmo\lib\site-packages (from cadati) (1.0.3)
Requirement already satisfied: pytz>=2017.2 in c:\users\wpreimes\appdata\local\continuum\miniconda2\envs\pytesmo\lib\site-packages (from pandas->cadati) (2020.1)
Requirement already satisfied: python-dateutil>=2.6.1 in c:\users\wpreimes\appdata\local\continuum\miniconda2\envs\pytesmo\lib\site-packages (from pandas->cadati) (2.8.1)
Requirement already satisfied: six>=1.5 in c:\users\wpreimes\appdata\local\continuum\miniconda2\envs\pytesmo\lib\site-packages (from python-dateutil>=2.6.1->pandas->cadati) (1.14.0)
